### DISCUSSION TITLE
Latest ver of Echo dots

### DISCUSSION THREAD

**Discussion by MirzFiz** [vintlabs/fauxmoESP#189] [Q&A]
Hi, i wanna clarify something regarding on echo dots ver. Does the newer ver echo dot such as 4th gen can be used together with this code/software.

**Comment by sgouedj** [comment]
hi MirzFiz,
do you have any answer?
i have the echo dot 4 and Alexa detect my esp8266 but it answer #server not responding#.
do you have the same thing?
thx
